webapp/graphql/PostQuery.js

```diff
@@ -81,6 +81,8 @@ export const filterPosts = (i18n) => {
           lng
           lat
         }
+        isObservedByMe
```

Comment:
I think this is not needed as it comes from the fragment

Reply:
You're right!